Lab1

mbernhard7 opened the pull request:
Submission of lab 1 for Miles and Vera. We would love to get any feedback, but we are conscious of the fact that it is the day before the deadline and you may not have time. Looking forward to your feedback regardless!

Commit:
first addition

Commit:
second pass

Commit:
third pass

Commit:
third pass

mbernhard7 committed:
working on edit page

mbernhard7 committed:
working on edit page

mbernhard7 committed:
Removing project junk

Commit:
fourth pass

Commit:
fifth pass

Commit:
5th pass

mbernhard7 committed:
Finalizing html files

Commit:
last pass

mbernhard7 committed:
Final commit (hopefully)

mbernhard7 committed:
Final commit (probably)

mbernhard7 committed:
Final commit

nrhodes-hmc reviewed (commented):
Nice job.

I like "All your tasks are done". Much nicer than just an empty page.

![Second Draft](./images/mockups/Second Draft.jpg) isn't displaying in design.md

I like the All/Uncompleted widget.

As you are showing the task flow, make sure you indicate what user action is causing the change from one screen to the next.
For example: In an empty list, create an item named "Buy new John Grisham book"
Screen at the beginning of task

You need to explicitly state that clicking on the + and then typing "Buy ... book" leads to the next screen.

Unclear what Save button does when in Edit mode. Is that the only way to get out of Edit mode? Is there a way to leave but not save?
If Save is the only way to get out, then it sounds like you're just going back to non-edit mode.  Maybe a close box instead?

Commit:
fixes to design.md

Commit:
updates to images and design file

mbernhard7 committed:
Fixing checkboxes

mbernhard7 commented:
> Nice job.
> 
> I like "All your tasks are done". Much nicer than just an empty page.
> 
> ![Second Draft](./images/mockups/Second Draft.jpg) isn't displaying in design.md
> 
> I like the All/Uncompleted widget.
> 
> As you are showing the task flow, make sure you indicate what user action is causing the change from one screen to the next.
> For example: In an empty list, create an item named "Buy new John Grisham book"
> Screen at the beginning of task
> 
> You need to explicitly state that clicking on the + and then typing "Buy ... book" leads to the next screen.
> 
> Unclear what Save button does when in Edit mode. Is that the only way to get out of Edit mode? Is there a way to leave but not save?
> If Save is the only way to get out, then it sounds like you're just going back to non-edit mode. Maybe a close box instead?

Thanks for your feedback Prof. Rhodes. We just updated the design.md file and made a few changes to our interface. Namely, we changed the Save button to a Done button on the edit page to imply that clicking it will mean you are done editing, therefore returning to the normal screen. We figure that during lab2 with implementation, we can add alerts with javascript to confirm the user really wants to do actions like deleting all or applying edits.

nrhodes-hmc commented:
* Thoughtful design iterations
* Thoughtful testing
